### DISCUSSION TITLE
Playing the game but it doesn't save the time played

### DISCUSSION THREAD

**Discussion by Hackerprod** [SteamRE/SteamKit#1218] [General]
Hello, I am implementing an application where I open a game with the code:

            var playGame = new ClientMsgProtobuf<CMsgClientGamesPlayed>(EMsg.ClientGamesPlayed);

            playGame.Body.games_played.Add(new CMsgClientGamesPlayed.GamePlayed
            {
                game_id = new GameID(appId), // or game_id = APPID,
            });

in other users it appears as playing but the game played time in the account does not increase, am I forgetting something?
Greetings

**Comment by voided** [comment]
I had replied on IRC but you dropped off:

> in the past that was usually all you needed
  best bet would be to check out the ArchiSteamFarm project and see what they use, cause i imagine it sends that message to increase time
https://github.com/JustArchiNET/ArchiSteamFarm/blob/089cc5beee7db0630cf058c74d0751d25d6d6490/ArchiSteamFarm/Steam/Integration/ArchiHandler.cs#L494 this seems like the relevant code
